Python/Product/PythonTools/PythonTools/Intellisense/FuzzyCompletionSet.cs

```diff
@@ -221,6 +223,8 @@ public class FuzzyCompletionSet : CompletionSet {
                 }
                 _filteredCompletions.Filter(IsVisible);
             }
+
+            CommitByDefault = DefaultCommitByDefault;

```

Comment:
Considering we don't publish the XML docs anywhere, the only place you'll find them is in IntelliSense or by reading the source code. But I'll add it and refer to the constant to cover all bases.